prow/github/client.go

```diff
@@ -617,7 +617,7 @@ func (c *Client) AssignIssue(org, repo string, number int, logins []string) erro
 	}
```

Comment:
`assignee.Login` should be wrapped in the normalization func.

Reply:
done.